vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -377,6 +377,8 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			resourcePath = itemPath
 			resourceParams = nameParams
 		}
+		apiResource.Group = a.group.GroupVersion.Group
```

Comment:
these are only normally set if they are different.  I suppose its not strictly wrong though.